File: TorrentTracker/Models/DTO/PeerDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TorrentTracker.Models.DTO
{
    public class PeerDTO
    {
        public string ID { get; set; }
        public bool Available { get; set; }
        public bool CorrectSendData { get; set; }
        public List<string> BannedPeers { get; set; }
        public List<FileDTO> Files { get; set; }
    }
}

File: sob-gui-master/sob-gui-master/src/app/dto/PeerDTO.ts
import { Guid } from "guid-typescript";
import { FileDTO } from "./FileDTO";

export class PeerDTO {
    id: string;
    available: boolean;
    correctSendData: boolean;
    bannedPeers: Guid[];
    files: FileDTO[];


    constructor(id: string, available: boolean, correctSendData: boolean, bannedPeers: Guid[], files: FileDTO[]) {
        this.id = id;
        this.available = available;
        this.correctSendData = correctSendData;
        this.bannedPeers = bannedPeers;
        this.files = files;
    }

}